.README/rules/check-tag-names.md

```diff
@@ -75,10 +75,28 @@ version
 yields
 ```
 
+#### Options
+
+##### `definedTags`
```

Comment:
How about `customTags` or `additionalTags`? `definedTags` is fine though.

Reply:
I considered this as well, but I thought it'd be more consistent with `definedTypes` (used in the `no-undefined-types` rule).